cmd/tdexd/main.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/tdex-network/tdex-daemon/internal/core/ports"
 	oceanwallet "github.com/tdex-network/tdex-daemon/internal/infrastructure/ocean-wallet"
 	webhookpubsub "github.com/tdex-network/tdex-daemon/internal/infrastructure/pubsub/webhook"
+	swap_parser "github.com/tdex-network/tdex-daemon/internal/infrastructure/swap-parser"
```

Comment:
```suggestion
	swapparser "github.com/tdex-network/tdex-daemon/internal/infrastructure/swap-parser"
```

just to be consistent?